cpp/include/cudf/detail/utilities/hash_functions.cuh

```diff
@@ -28,71 +28,88 @@ using hash_value_type = uint32_t;
 
 namespace cudf {
 namespace detail {
-
 /**
- * @brief Core MD5 algorithm implementation. Processes a single 512-bit chunk,
- * updating the hash value so far. Does not zero out the buffer contents.
+ * Modified GPU implementation of
+ * https://johnnylee-sde.github.io/Fast-unsigned-integer-to-hex-string/ that is lowercase only and
```

Comment:
You can't use someone else's code unless it is properly licensed or with their express, written permission.

Reply:
Has this been resolved?